src/components/UserSelector/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { DropdownItem } from './DropdownItem';
+import { User } from '../../../types/User';
+
+interface DropdownMenuProps {
+  users: User[];
+  setIsActive: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const DropdownMenu: React.FC<DropdownMenuProps> = ({
+  users,
+  setIsActive,
+}) => {
+  return (
+    <div tabIndex={1} className="dropdown-menu" id="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {users.map(user => (
+          <DropdownItem key={user.id} user={user} setIsActive={setIsActive} />
```

Comment:
Avoid generating keys during render using properties like `user.id`. If `user.id` is not guaranteed to be unique or stable, consider using a different unique identifier.